packages/strapi-plugin-graphql/services/GraphQL.js

```diff
@@ -486,7 +486,7 @@ module.exports = {
 
       // Detect enum and generate it for the schema definition
       const enums = Object.keys(model.attributes)
-        .filter(definition => definition.type === 'enumeration')
+        .filter(attribute => model.attributes[attribute].type === 'enumeration')
```

Comment:
What do you think about this lodash's version ?

```JS
const enums = _.chain(model.attributes)
  .filter({ type: 'enumeration' })
  .map((definition, attribute) => {
    ...
  })
  .value()
````

Reply:
@lauriejim @Aurelsicoko thoughts? I don't know how this bit functions to debug it.

Reply:
@kamalbennani as said @Aurelsicoko (cf. https://github.com/strapi/strapi/pull/1275#pullrequestreview-123560796), I share with him the goal to not use lodash because ES6 and next are really efficient now and it allow to use "simpler" functions to iterate some data (and it also avoid some of us to go to the lodash docs to know what make some mysterious functions).